SPECS/xapi.spec

```diff
@@ -1028,7 +1049,7 @@ Coverage files from unit tests
 - CA-258385: Improved phrasing for errors thrown by assert_can_migrate.
 - Replace /tmp/network-reset literal with Xapi_globs
 - Travis: remove opam-coverage
-- Modifications to the error messages for better compliance with the 
+- Modifications to the error messages for better compliance with the
```

Comment:
Unexpected change (trailing space removed). Many more below.

Reply:
I saw but since it's not designed to be merged i thought it was okay to leaeve it.

Reply:
Ok